### ISSUE TITLE
Invariant violation on IE

### ISSUE THREAD

**Issue by shankie-san** [outlandishideas/kasia#76]
I'm having some fun with IE – on every version of IE that I can get my hands on, I get the following error:

`Invariant violation: Connect(KasiaConnectWpQueryComponent).render(): A valid React element (or null) must be returned`

Classic 😁  . Before I produce a minimal test case, can anyone else confirm if they've got this error?

**Comment by sdgluck** [comment]
Thanks for reporting this @shankie-san. I haven't seen this myself so would appreciate seeing a repro if you get the time.

**Comment by shankie-san** [comment]
Hey @sdgluck it turns out that this was operator error, well at least partially. IE11 doesn't have `fetch` built in (and I think `wpapi` uses fetch), so I had to polyfill it using webpack. So, not specifically a kasia issue, but might be worth just a quick note in the docs that you'll have to polyfill fetch for IE11. Does that make sense.

Nice sighthound avatar, BTW. I've got a nervous greyhound sitting next to me at the moment 😁
